docs/advanced-guides/using-development-build.md

```diff
@@ -52,7 +52,23 @@ Next make the `ignite` command available globally. To do this, run `yarn link`:
 yarn link
 ```
 
-> Note: If you get an error at this point, it is probably related to not removing a previously installed version of Ignite CLI. See the commands above to uninstall Ignite CLI.
+> Note: If you get an error at this point, it is probably related to not removing a previously installed version of Ignite CLI. See the commands above to uninstall Ignite CLI. 
+
+> Note: If you have run through the `npm rm` commands listed above and you get this error upon running `yarn link`
+
+  ```sh
+  warning There's already a package called "ignite-cli" registered. This command has had no effect. If this command was run in another folder with the same name, the other folder is still linked. Please run yarn unlink in the other folder if you want to register this folder.
+  ```
+
+  You may have a symlink that must be removed. CD to `~/.config/yarn/link` and run `ls`
+
+  If you see a symlink listed as `ignite-cli`, delete it. CD back to the `ignite` directory and run `yarn link`. It should run without errors and return something similar to the following in the terminal:
+
+  ```sh
```

Comment:
The `sh` here is messing up the formatting on you.  I'd say drop that since this output aren't shell commands.